plutus-ledger-api/test/Spec/CostModelParams.hs

```diff
@@ -30,13 +30,13 @@ tests =
             166 @=? length v1CostModelParamNames
             175 @=? length (enumerate @V2.ParamName)
             175 @=? length v2CostModelParamNames
-            217 @=? length (enumerate @V3.ParamName)
-            217 @=? length v3CostModelParamNames
+            223 @=? length (enumerate @V3.ParamName)
+            223 @=? length v3CostModelParamNames
     , testCase "context length" $ do
             let defaultCostValues = Map.elems $ fromJust defaultCostModelParams
             -- the defaultcostmodelparams reflects only the latest version V3, so this should succeed because the lengths match
             assertBool "wrong number of arguments in V2.mkContext" $ isRight $ runExcept $ runWriterT $ V3.mkEvaluationContext defaultCostValues
-            -- currently v2 args ==v3 args
+            -- currently v2 args == v3 args
```

Comment:
Something is funny about these tests, should one of these be failing? @bezirg

Reply:
I think PLT-5290 (which we were talking about yesterday) should cover that.  I had to add some stuff to those tests for BLS12-381 a while ago and I wasn't totally confident that I'd got it right.

Reply:
Oh, that comment's wrong.  I corrected the spacing but I should probably just have deleted it.